readinglist/views/article.py

```diff
@@ -41,12 +41,15 @@ class ArticleSchema(ResourceSchema):
     added_on = TimeStamp()
     stored_on = TimeStamp()
 
-    status = SchemaNode(colander.Integer(), missing=0)
+    status = SchemaNode(colander.Integer(), missing=0,
+                        validator=colander.Range(min=0, max=1))
     favorite = SchemaNode(colander.Boolean(), missing=False)
     unread = SchemaNode(colander.Boolean(), missing=True)
     is_article = SchemaNode(colander.Boolean(), missing=True)
     excerpt = SchemaNode(String(), missing="")
 
+    read_position = SchemaNode(colander.Integer(), missing=0,
+                               validator=colander.Range(min=0))
```

Comment:
we don't have any max number of words (even theoretical?) I believe we should :)